recipes/full_finetune_distributed.py

```diff
@@ -559,6 +564,9 @@ def _setup_model(
                 parallelize_plan=self.tp_plan,
             )
 
+        if self._enable_fp8_training:
+            model = convert_to_float8_training(model)
```

Comment:
Noob clarification q but based on [this comment block](https://github.com/pytorch/ao/blob/c9b1490971ab3b95c20f076e8dd3a679e27883dc/torchao/float8/fsdp_utils.py#L96-L113) in ao it would seem that float8 conversion occurs prior to TP parallelization. Just wanna confirm that this isn't necessary (iiuc from how titan does it the two are basically done concurrently)

Reply:
I think the current state in this PR does them in the right order as described in the comment block you linked?